tests/test_storage.py

```
@@ -77,6 +77,70 @@ async def test_logs_allows_valid_paths(self, _mock_graphql: AsyncMock) -> None:
         result = await tool_fn(action="logs", log_path="/var/log/syslog")
         assert result["content"] == "ok"
 
+    async def test_logs_tail_lines_too_large(self, _mock_graphql: AsyncMock) -> None:
```

Ruff (0.15.1)

[warning] Fixture `_mock_graphql` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)